Services/UserService.cs

```diff
@@ -299,6 +306,124 @@ public async Task<ModifyUserResponse> ModifyUser(ModifyUserRequest rq, ClaimsIde
                 };
             }
         }
+
+        public async Task<DeleteUserResponse> DeleteUser(DeleteUserRequest rq, ClaimsIdentity? identity)
+        {
+            var token = identity.TokenVerifier();
+            var validation = ValidateDeleteUser(rq, token);
+
+            if (validation != String.Empty)
+            {
+                return new DeleteUserResponse
+                {
+                    Message = validation,
+                    Success = false,
+                };
+            }
+
+            var user = await _context.User
+                .Where(x => x.UserId == rq.UserId)
+                .FirstOrDefaultAsync();
+
+            if (user == null)
+            {
+                return new DeleteUserResponse
+                {
+                    Success = false,
+                    Message = "User not found",
+                };
+            }
+
+            if (user.Bills.Count > 0 ||
+                user.Logs.Count > 0 ||
+                user.Messages.Count > 0 ||
+                user.Purchases.Count > 0 ||
+                user.Sales.Count > 0)
+            {
+                user.Inactive = true;
+            }
+            else
+            {
+                _context.User.Remove(user);
+            }
+
+            try
+            {
+                await _context.SaveChangesAsync();
+                return new DeleteUserResponse
+                {
+                    Success = true,
+                    Message = "Succesfully user deleted",
+                };
+            }
+            catch (Exception ex)
+            {
+                return new DeleteUserResponse
+                {
+                    Success = false,
+                    Message = ex.Message,
+                };
+            }
+        }
+
+        public async Task<ActivateUserResponse>ActivateUser(ActivateUserRequest rq, ClaimsIdentity ? identity)
```

Comment:
Add space before and after Activate user, remove space in the ? on ClaimsIdentity